### ISSUE TITLE
EtherScanInfo should have a public initializer

### ISSUE THREAD

**Issue by mousehaohao** [essentiaone/essentia-bridges-api-ios#27]
Hi Team,
    when I use this pod library, the default initializer is not work for etherscaninfo, the error message is "'EtherScanInfo' initializer is inaccessible due to 'internal' protection level". so I think it should be have a public initializer for this class.
     this code seems works:

   ```
public struct EtherScanInfo {
    public init(url: String, apiKey: String) {
        self.url = url
        self.apiKey = apiKey
    }
    public let url: String
    public let apiKey: String
}
```

**Comment by impul** [comment]
Hi, @mousehaohao.
You are right. I will fix this asap.
For now you can use `CryptoWallet` class
Example: 
```
let cryptoWallet = CryptoWallet(
            bridgeApiUr: bridgeUrl,
            etherScanApiKey: apiKey)
```
And then you will be able to use 
```
cryptoWallet.ethereum
```

**Comment by impul** [comment]
@mousehaohao, fixed at [0.4.5](https://github.com/essentiaone/essentia-bridges-api-ios/commit/da71023c4564c934f24a6f7de53e41f891c86be3)
